juju/juju
# Azure: Use standard sku Public IP addresses

## Description of change

As per the bug below, use the standard sku for any public IP address that is created (this allows the use of the standard load balancer etc.)

This branch also updates the azure SKU used to the newest version.


## QA steps

When you deploy any machines to an Azure provider, using the Azure we UI inspect the created public IP addresses in the resources viewer and see the SKU states 'standard' (as opposed to 'basic').

## Documentation changes
 n/a

## Bug reference

https://bugs.launchpad.net/juju/+bug/1799238
